.pylintrc

```diff
@@ -212,7 +213,7 @@ max-nested-blocks=5
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=105
```

Comment:
Is there a significance to 105? Just curious.

Reply:
It accomodates the fact that default the line length limit in Black is not a hard limit. This change makes sure that linting doesn't fail with:

```
************* Module qiskit_ibm_runtime.credentials.hub_group_project_id
qiskit_ibm_runtime/credentials/hub_group_project_id.py:49:0: C0301: Line too long (102/100) (line-too-long)
```

Background: 
- https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
> [...] Black defaults to 88 characters per line [...] 
> Black will try to respect that. However, sometimes it won’t be able to without breaking other rules. In those rare cases, **auto-formatted code will exceed your allotted limit.**